pkg/app/piped/cloudprovider/lambda/client.go

```diff
@@ -66,10 +66,7 @@ func newClient(region, profile, credentialsFile string, logger *zap.Logger) (*cl
 	return c, nil
 }
 
-func (c *client) Apply(ctx context.Context, fm FunctionManifest, role string) error {
-	if role == "" {
-		return fmt.Errorf("role arn is required")
-	}
```

Comment:
I've not tested but I'm curious about is it ok to give the address to an empty string if the role not given. I mean don't we have to explicitly give nil to `CreateFunctionInput`? Just out of curiosity, never mind if no problem.

Reply:
> is it ok to give the address to an empty string if the role not given.

I think it's okay, but since the current fields of `CreateFunctionInput` are all required fields ( I mean we have to pass not nil value or we will get InvalidParameterValueException ), it makes no sense to pass nil value as CreateFunction input.
On the other hand, the reason why I remove this check condition is though we pass an empty string role pointer here, `lambda.ErrCodeInvalidParameterValueException` will be raised by the validation step of `CreateFunction`, so no need to redundancy check it here. Sorry for my lack of sharing unusual remove code. 🙏

Reply:
That makes total sense. Thank you for letting me know!